.taskcluster.yml

```diff
@@ -0,0 +1,26 @@
+version: 1
+policy:
+  pullRequests: public
+tasks:
+  $match:
+    '(tasks_for == "github-pull-request" && event["action"] in ["opened","reopened","synchronize"])':
+      taskId:
+        $eval: as_slugid("pr_task")
+      provisionerId: proj-getting-started
+      workerType: tutorial
```

Comment:
Hm, I imagine we need to change this too, but I'm not sure what to. I'll ping someone who should know.